### ISSUE TITLE
Coridian: display issues on Linux - selected element in TV shows list becomes invisible

### ISSUE THREAD

**Issue by msrex** [Komet/MediaElch#1412]
**Describe the bug**
After upgrading to Coridian the active element in the TV shows list becomes invisible (maybe white on white?). This is true for any element, whether show name, season name or episode name. The movie list is _not_ affected and always displays normally.

**To Reproduce**
1. Start MediaElch
2. Display TV show list
3. Select any item
4. It disappears

**Expected behavior**
The selected item should remain visible

**Screenshots**
img1 shows TV show list after starting MediaElch
img2 has the show name selected (clicked on Bad Banks)
img3 has the season selected (in this case season 2)
img4 has the episode selected (in this case episode 3 - Kollateralschaden)

**MediaElch Version:**
 - [ ] 2.8.15-dev (nightly)
 - [X] 2.8.14 stable

**Operating System:**
 - [ ] Windows
 - [ ] macOS
 - [X] Linux (distribution: openSUSE 15.3 - installed from home:bugwelle)

**Additional context**
With 2.8.12 on the same machine the display is normal. I cloned the bugwelle OBS project to get back to 2.8.12 to confirm.

![img1](https://user-images.githubusercontent.com/2625328/153760147-4c9b8abb-d9cb-45a9-93c3-93dc8c4b6b1b.png)
![img2](https://user-images.githubusercontent.com/2625328/153760149-3e123613-128b-4e58-ae1c-527f899d6c38.png)
![img3](https://user-images.githubusercontent.com/2625328/153760150-1c61075d-8aa6-4888-852c-f90dafc80614.png)
![img4](https://user-images.githubusercontent.com/2625328/153760151-e7729f17-9c3e-41fa-8b2f-a1e580bb630f.png)

**Comment by JalilArfaoui** [comment]
Same issue here 👋🏾

**Comment by bugwelle** [comment]
Aaarrghhh... I now exactly what you mean. I have this with the non-system Qt version as well. It's some style that gets applied. I never got to debug it properly...

Thanks for bringing this up.

**Comment by bugwelle** [comment]
Before:
![Screenshot_20220219_180104](https://user-images.githubusercontent.com/1667306/154811786-e24f7191-5ae4-4784-8a7c-f9dafb57317e.png)


After:
![Screenshot_20220219_182044](https://user-images.githubusercontent.com/1667306/154811790-0fcb43d7-8fde-4873-a066-0a581e02c9e2.png)

Will be fixed by #1420

**Comment by Rygle** [comment]
Can you please do a 2.8.14.1 build just for this bug? I'm glad you've fixed it, but it's still the only build that is available in the ppa. Anyone on a Debian/Ubuntu distro is stuck with this painful bug until you update the ppa.

Thanks!

**Comment by bugwelle** [comment]
Will do. See https://github.com/Komet/MediaElch/discussions/1425#discussioncomment-2308070 :-)

**Comment by workflowsguy** [comment]
The same issue also exists on macOS

**Comment by bugwelle** [comment]
Related changed with #1524

**Comment by phpcitizen** [comment]
I have the same issue now on the latest [Version 2.10.0 "Benzar"](https://github.com/Komet/MediaElch/releases/tag/v2.10.0)

I am on Nixos.

**Comment by Rygle** [comment]
Hi,

I don't seem to have had this problem with recent builds. Perhaps as Nixos is quite a niche/specialised system, you might have missed including some dependencies that MediaElch needs to run? Not sure...

Perhaps one idea (that is not quite as good as running a native client, but might be a suitable workaround, or a test setup to compare against your native client) is to try a Docker image? I have made a Docker image with the most recent build at https://hub.docker.com/r/ozrygle/mediaelch

There are instructions to run it using Docker on that page.

Basically you should be able to install Docker, then run;
```
docker pull ozrygle/mediaelch

docker run -p 5800:5800 -p 5900:5900 -v /path/to/config/folder:/config/xdg/config/kvibes -v /path/to/media:/media/ -e PGID=1000 -e PUID=1000 mediaelch:latest
```
If course in the above command you can have extra media paths  "-v /path/to/media/tv:/media/tv
You should then be able to log in to MediaElch using a web browser at the IP address of the Docker host and the port number you set (e.g. 127.0.0.1:5800 if on the local computer)

**Comment by bugwelle** [comment]
It could be that default colors are leaking through MediaElch's main window UI. For example, if there is a line color that isn't specified explicitly, the system's line color will be used.
